location/utils.py

```diff
@@ -54,3 +54,48 @@ def approx_home_location(data, sr_col='stay_region'):
     home = Counter(trg_data[sr_col]).most_common()[0][0]
 
     return home
+
+
+def compute_total_gyration(data, sr_col='stay_region'):
+    """
+    Computes the total radius of gyration.
+
```

Comment:
I would recommend you explain the algorithm here in details so that others find it easier to follow.